apps/authentication/templates/authentication/face_capture.html

```diff
@@ -35,7 +35,7 @@
     <script>
         $(document).ready(function () {
             const apiUrl = "{% url 'api-auth:mfa-face-context' %}";
-            const faceCaptureUrl = "/faceliving/capture";
+            const faceCaptureUrl = "/facelive/capture";
             let token;
 
             function createFaceCaptureToken() {
```

Comment:
None detected. The code appears to be valid Python. Any regular checks would depend on specific context that the code was generated against, which we could not access at this time due to the knowledge cutoff limit mentioned previously.